userspace/Build.mk

```diff
@@ -12,9 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-BUILD_SUBDIRS := $(addprefix userspace/,aes_test dcrypto_test flash_test gpio_test \
-		low_level_debug nvcounter_test nvcounter_ctest personality_clear \
-		personality_test sha_test spin u2f_app u2f_test )
+BUILD_SUBDIRS := $(addprefix userspace/,                   \
+					 aes_test          \
+					 blink             \
+					 dcrypto_test      \
+					 flash_test        \
+					 gpio_test         \
+					 low_level_debug   \
+					 nvcounter_ctest   \
+					 nvcounter_test    \
+					 personality_clear \
+					 personality_test  \
+					 rng               \
+					 sha_test          \
+					 spin              \
+					 u2f_app           \
+					 u2f_test )
```

Comment:
The alignment here is funny. GitHub renders these tabs as 8-wide. Does aligning with spaces work? I don't know Make's syntax well enough.

Reply:
Weird -- I actually manually spaced them. I'm wondering if emacs did the annoying "turn spaces to tabs" thing. I'll fix it.